apps/www/_blog/2024-06-18-calcom-platform-starter-kit-nextjs-supabase.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: 'Cal.com launches Expert Marketplace built with Next.js and Supabase.'
+description: Cal.com and Supabase team up to build an open-source platform starter kit for developers.
+author:
+image:
+thumb:
+categories:
+  - developers
+tags:
+  - open-source
+  - partnerships
+  - nextjs
+date: '2024-06-18'
+toc_depth: 2
+---
+
+Contrary to popular belief, due to the biggest online beef in modern history, Calcom and Supabase are actually supa good friends, united by a common mission to build open source software.
+
+![ze germans tweet]()
+
+So when the calcom team reached out about collaborating on on their new platform starter kit, we were excited to work together. Finally we could collaborate on a ProductHunt launch instead of competing against each other.
```

Comment:
Cal.com

Reply:
```suggestion
So when the Cal.com team reached out about collaborating on on their new platform starter kit, we were excited to work together. Finally we could collaborate on a ProductHunt launch instead of competing against each other.
```